zeroex/order_validator.go

```diff
@@ -167,6 +167,10 @@ var (
 		Code:    "OrderHasInvalidSignature",
 		Message: "order signature must be valid",
 	}
+	ROMaxExpirationExceeded = RejectedOrderStatus{
```

Comment:
@fabioberger It doesn't exactly feel right to move `ROMaxExpirationExceeded` and `MeshValidation` into this package but I couldn't figure out another way to do it without introducing cyclical imports. Any ideas?

Reply:
Agreed. We could also create a dedicated package but this works for now.